Document known security integration issue (#7125)
elastic/docs-content: release-notes/elastic-security/known-issues.md

% :::

:::{dropdown} Problem Child and DGA integrations fail to install

> **Applies to: {{stack}} 9.3.6 and 8.19.17**

Reviewer: ```suggestion
**Applies to: {{stack}} 9.3.6**
```

**Impact**<br>

> The [Problem Child](https://www.elastic.co/docs/reference/integrations/problemchild) and [DGA](https://www.elastic.co/docs/reference/integrations/dga) integration packages fail to install. This is caused by an {{es}} validation that limits field names in trained models to 100 characters.

Reviewer 2: ```suggestion
The [ProblemChild](https://www.elastic.co/docs/reference/integrations/problemchild) (Living off the Land Detection) and [DGA](https://www.elastic.co/docs/reference/integrations/dga) integration packages fail to install. This is caused by an {{es}} validation that limits field names in trained models to 100 characters.
```

**Workaround**<br>

> Upgrade to {{stack}} 9.3.7 or 8.19.18.

Reviewer: ```suggestion
Upgrade to {{stack}} 9.3.7.
```

**Resolved**<br>

> Resolved in {{stack}} 9.3.7 and 8.19.18.

Reviewer: ```suggestion
Resolved in {{stack}} 9.3.7.
```
